### ISSUE TITLE
Breaks when I add 50/50 custom zones

### ISSUE THREAD

**Issue by ThePolitePunk** [gerritdevriese/kzones#88]
Hi, I love this addon. I'm not on an ultrawide, but it's useful because I have windows set to fly over to the other virtual desktop when I drag them, so edge snapping is awkward. Plus I like having more options of splitting modes.

Anyway, since I'm not on an ultrawide, I tried writing a 50/50 split into the 'advanced.json' example as follows.

` {
        "name": "Split Screen",
        "padding": 0,
        "zones": [
            {
                "x": 0,
                "y": 0,
                "height": 100,
                "width": 50
            },
            {
                "x": 50,
                "y": 0,
                "height": 100,
                "width": 50,
            }
        ]
    }`

I've tried adding it at the bottom and the top, I'll add the whole .json on the next post for clarity.

Basically, when I add this to the layouts tab, KZones just stops working. I can't switch layout type, I can't snap windows, nothing. And this can only be fixed by reinstalling the program and pasting in a default .json, even editing it back to how it was or using the reset function doesn't work. Sometimes I have to restart the whole system for it to properly reset!

I'm not sure how I broke it so bad... I'm sure I'm not this bad at code!
